v1.1/certificate-rotation.md

```diff
@@ -96,26 +115,31 @@ To overcome the issue, we take advantage of the fact that multiple CA certificat
  
 ### Why rotate CA certificates in advance 
 
-Going back to the CA certificate rotation process, the second step of the process involves rotating the node and client certificates. This involves signing the node certificates with new CA certificates. These new node certificates are uploaded to the nodes as soon as the certs directory on the node is rescanned. But the clients will get their new client certificates signed with the new CA certificates only when they are restarted. The clients won't even have the new CA certificates till they are restarted. Thus node certificates signed by the new CA certificate will not be accepted by clients that do not have the new CA certificate yet. To ensure all nodes and clients have the latest CA certificate, change CA certificates on a completely different schedule, for instance, months before changing the node and client certificates. 
+Going back to the CA certificate rotation process, the second step of the process involves rotating the node and client certificates. This involves signing the node certificates with new CA certificates. These new node certificates are used by the nodes as soon as the certs directory on the node is rescanned. But the clients will get their new client certificates signed with the new CA certificates only when they are restarted. The clients won't even have the new CA certificates till they are restarted. Thus node certificates signed by the new CA certificate will not be accepted by clients that do not have the new CA certificate yet. To ensure all nodes and clients have the latest CA certificate, change CA certificates on a completely different schedule, for instance, months before changing the node and client certificates. 
+
+### Steps to rotate CA certificate
 
-## Steps to rotate CA certificate
+#### Step 1: Generate new CA key and certificate and append it to existing CA certificate:
 
-### Step 1: Generate new CA certificate and append it to existing CA certificate:
+Suppose your existing CA and node certificates are created in the `certs` directory and your CA key is stored in a directory called `my-safe-directory`. 
 
-To rotate CA certificates, use the `--overwrite` flag to generate a new CA certificate and key. 
+Then to rotate the CA certificate, first rename the existing CA key:
 
 ~~~ shell
-$ cockroach cert create-ca --certs-dir=certs --ca-key=ca.new.key --overwrite 
+$ mv ca.key ca.old.key
```

Comment:
We should point to the same path as below: `my-safe-directory/ca.key` and `my-safe-directory/ca.old.key`

Reply:
@mberhault Thank you! Although, I am not clear on why we need to do this. As in, why do we need to retain the old key when we do not yet support signing new node/client certificates with old  CA certificates?

Reply:
For one, because we will need the key once we allow signing with the old CA. But also because it's rarely a good idea to just delete a key completely.